src/apps/games/models/game_count_views.py

```diff
@@ -16,49 +16,56 @@ class GameCountByNetwork(pg.MaterializedView):
 
     sql = """
     SELECT
-      trainingcounts.*,
-      ratingcounts.total_num_rating_games as total_num_rating_games,
-      ratingcounts.total_rating_score as total_rating_score,
+      counts.*,
       trainings_network.name as network_name,
       trainings_network.run_id as run_id,
       trainings_network.created_at as network_created_at
-    FROM (
-      SELECT
-      black_network_id as network_id,
-      count(*) as total_num_training_games,
-      sum(num_training_rows) as total_num_training_rows
-      FROM games_traininggame
-      GROUP BY black_network_id
-    ) trainingcounts
-    INNER JOIN
+    FROM
     (
-      SELECT network_id, sum(num_games) as total_num_rating_games, sum(score) as total_rating_score
-      FROM
+      SELECT
+        COALESCE(trainingcounts.network_id, ratingcounts.network_id) as network_id,
+        COALESCE(trainingcounts.total_num_training_games, 0) as total_num_training_games,
+        COALESCE(trainingcounts.total_num_training_rows, 0) as total_num_training_rows,
+        COALESCE(ratingcounts.total_num_rating_games, 0) as total_num_rating_games,
+        COALESCE(ratingcounts.total_rating_score,0.0) as total_rating_score
+      FROM (
+        SELECT
+        black_network_id as network_id,
+        count(*) as total_num_training_games,
+        sum(num_training_rows) as total_num_training_rows
+        FROM games_traininggame
+        GROUP BY black_network_id
+      ) trainingcounts
+      FULL OUTER JOIN
       (
+        SELECT network_id, sum(num_games) as total_num_rating_games, sum(score) as total_rating_score
```

Comment:
CTE (having a select inside a query) is consuming some memory.

We may change the sql later (optimize it).